### PR TITLE
WIP: flughafen: add initial configuration for flughafen

### PR DESCRIPTION
**Pull request by booo**
Still needs a IPv6 prefix, snmp configuration and DFS magic.

Could someone please review my first interaction with this tool. Thanks a lot for your time!

### CODE REVIEW

File: host_vars/flughafen-core/base.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+
+location: fluehafen
```
Comment (PolynomialDivision) [inline host_vars/flughafen-core/base.yml:3]: fluehafen?

**Comment by PolynomialDivision** [comment]
Please have a look here, we agreed on vlan numbering:
https://github.com/Freifunk-Spalter/bbb-configs/issues/3#issuecomment-889040037

File: host_vars/flughafen-core/base.yml (hunk 1 above)
Comment (booo) [reply host_vars/flughafen-core/base.yml:3]: Should be floehhafen instead right?

**Comment by booo** [comment]
> Please have a look here, we agreed on vlan numbering: [#3 (comment)](https://github.com/Freifunk-Spalter/bbb-configs/issues/3#issuecomment-889040037)

I don't want to change the vlan ids. Sites with a few more VLANs will run into problems with the proposed scheme anyway.

**Comment by Akira25** [comment]
@PolynomialDivision Why is there already a location flughafen on masterbranch?

**Comment by PolynomialDivision** [comment]
@booo said that we should finalize the flughafen because he has no time.

**Comment by booo** [comment]
Did you use the configuration from here?

Why has flughafen now a totally different IP configuration?

**Pull request closed without merging**